Catappult/appcoins-wallet-android
# [APPC-4053] Remove billing address code

**What does this PR do?**

   Since the feature was disabled on microservices side, we should remove the unused code

**Database changed?**

 No

**How should this be manually tested?**

  Try to make a IAP/Onboarding payment/Top-up with VPN to USA with a new credit card, you should be redirected to verify wallet instead of filling billing address

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4053](https://aptoide.atlassian.net/browse/APPC-4053)

**Code Review Checklist**



[APPC-4053]: https://aptoide.atlassian.net/browse/APPC-4053?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
